RecoEgamma/ElectronIdentification/src/ElectronMVAEstimatorRun2Spring16GeneralPurpose.cc

```diff
@@ -0,0 +1,460 @@
+#include "RecoEgamma/ElectronIdentification/interface/ElectronMVAEstimatorRun2Spring16GeneralPurpose.h"
+
+#include "DataFormats/TrackReco/interface/Track.h"
+#include "DataFormats/GsfTrackReco/interface/GsfTrack.h"
+
+#include "DataFormats/PatCandidates/interface/Electron.h"
+
+#include "FWCore/ParameterSet/interface/FileInPath.h"
+
+#include "TMath.h"
+#include "TMVA/MethodBDT.h"
+
+ElectronMVAEstimatorRun2Spring16GeneralPurpose::ElectronMVAEstimatorRun2Spring16GeneralPurpose(const edm::ParameterSet& conf):
+  AnyMVAEstimatorRun2Base(conf),
+  _tag(conf.getParameter<std::string>("mvaTag")),
```

Comment:
Variable names starting with "_" violate the CMS coding rules. They need to be changed.